src/Our.Umbraco.Vorto/Web/Controllers/VortoApiController.cs

```diff
@@ -46,6 +46,7 @@ public object GetDataTypeByAlias(string contentType, string contentTypeAlias, st
                 case "member":
                     ct = Services.MemberTypeService.Get(contentTypeAlias);
                     break;
+                case "settings":
```

Comment:
Any suggestions on how to get the setting type? There doesn't appear to be a SettingTypeService or similar.